Tensile/TensileCreateLibrary.py

```diff
@@ -244,7 +244,7 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
         archFlags = ["--offload-arch=" + arch for arch in cmdlineArchs]
 
         # needs to be fixed when Maneesh's change is made available
-        hipFlags = ["-D__HIP_HCC_COMPAT_MODE__=1"]
+        hipFlags = ["-D__HIP_HCC_COMPAT_MODE__=1", "-v"]
```

Comment:
My expectation was that this would output:

gfx900
gfx90a
etc

as we begin to build for those targets. Is this the case?

Reply:
I don't see this output in the rocBLAS build output, no.